src/java_tools/buildjar/java/com/google/devtools/build/buildjar/javac/plugins/errorprone/ErrorPronePlugin.java

```diff
@@ -121,6 +127,9 @@ public void postFlow(Env<AttrContext> env) {
     elapsed.start();
     try {
       errorProneAnalyzer.finished(new TaskEvent(Kind.ANALYZE, env.toplevel, env.enclClass.sym));
+      if (refactoringTask != null) {
+        refactoringTask.finished(new TaskEvent(Kind.GENERATE, env.toplevel, env.enclClass.sym));
```

Comment:
With this call here, the refactoring task won't actually run after the `GENERATE` phase is completed.  Assuming that it is important to run the task at that time, an alternative is to register the task as a `TaskListener` with the compiler.  To do this, around line 115 where the `RefactoringTask` is created, you can do:
```java
MultiTaskListener.instance(context).add(refactoringTask);
```
Then you won't need this call.  Not sure if it's important or not.

Reply:
Thanks @msridhar, I implemented your suggestion.